pdc/apps/common/constants.py

```diff
@@ -21,3 +21,9 @@
 
 # PDC warning field in response header
 PDC_WARNING_HEADER_NAME = 'pdc-warning'
+
+# Warning when using put method on optional parameter
+PUT_OPTIONAL_PARAM_WARNING = {
+    'PUT_OPTIONAL_PARAM_WARNING': 'When using the `PUT` method, if an optional field is not specified' +
+                                  ' in the input, it will be erased.',
```

Comment:
i think the original text has the meaning that "if the optional field has a default value, it will be reset to it", but, yes, maybe your description is more clear. @lubomir is that ok to change the description in this way "it will be erased" -> "it will be changed back to default value"